mne/channels/tests/test_montage.py

```diff
@@ -1663,3 +1665,19 @@ def test_make_wrong_dig_montage():
         make_dig_montage(ch_pos={'A1': ['a', 'b', 'c']})
     with pytest.raises(TypeError, match="instance of ndarray, list, or tuple"):
         make_dig_montage(ch_pos={'A1': 5})
+
+
+@testing.requires_testing_data
+def test_fnirs_montage():
+    """Ensure fNIRS montages can be get and set."""
+    raw = read_raw_nirx(fnirs_dname)
+    info_orig = raw.copy().info
+    mtg = raw.get_montage()
```

Comment:
can you check that mtg as the right number of channels? I assume twice the number of channels are you have sources and detectors?

Reply:
Based on this PR https://github.com/mne-tools/mne-testing-data/pull/51 we can tell that the montage has 5 sources, 13 detectors, and (8 x 2) = 16 channels.

Based on this PR the function will only return information for the 5 sources and 13 detectors. It does not return the channel based information (as that is extracted from the source and detector by `_set_montage_fnirs`).

I have added a test to ensure the correct number of sources and detectors is returned by `get_montage`. But we can change this if we take a different implementation approach to what I suggest.